### PR TITLE
docs: consolidate Template Engine Instances

### PR DESCRIPTION
**Pull request by adepanges**

### CODE REVIEW

File: docs/docs/templating.md [hunk 1]
```diff
@@ -53,6 +53,42 @@ export interface PlatformViewsSettings {
 ```
 
 ## Usage
+## Template Engine Instances
+
+Template engines are exposed via the `PlatformViews.consolidate.requires` object, but they are not instantiated until you've called the `PlatformViews.consolidate[engine].render()` method. You can instantiate them manually beforehand if you want to add filters, globals, mixins, or other engine features.
+
+::: Reference
```
Comment (Romakita) [inline docs/docs/templating.md:60]:
```typescript
::: tip Reference 
```

File: docs/docs/templating.md [hunk 2]
```diff
@@ -53,6 +53,42 @@ export interface PlatformViewsSettings {
...
+[Template Engine Instances](https://github.com/tj/consolidate.js#template-engine-instances).
+:::
+
+```js
```
Comment (Romakita) [inline docs/docs/templating.md:64]: \`\`\`typescript

**Review by Romakita — CHANGES_REQUESTED** [review]

File: docs/docs/templating.md (hunk 2 above)
Comment (adepanges) [reply docs/docs/templating.md]: already set as your feedback, is fine run `yarn test` from my local

error `@MultipartFile()` come from my changes?

**Comment by adepanges** [comment]
already set as your feedback, is fine run `yarn test` from my local

error `@MultipartFile()` come from my changes?

**Comment by Romakita** [comment]
You haven't rebased/started your branch from the latest modification  ;)

**Review by Romakita — APPROVED** [review]

**Pull request merged**